README.md

```diff
@@ -110,6 +111,23 @@ fn f(input: usize) -> usize {
 }
 ```
 
+# dont cache errors
```

Comment:
I'm not satisfied with this solution, but this was the only way I could figure out of passing `tests/readme.rs`. Just escaping the underscores, like `dont\_cache\_errors` did not seem to get the parser to understand this as one span.